distribution/lib/Standard/Table/0.0.0-dev/src/Delimited/Delimited_Format.enso

```diff
@@ -169,14 +170,17 @@ Delimited_Format.from (that : JS_Object) =
     encoding = encoding_name
         . if_not_nothing (Encoding.from_name encoding_name)
         . if_nothing Encoding.utf_8
-    headers = that.get "headers" . if_nothing Infer
+    headers = that.get "headers" |> parse_boolean_with_infer "headers"
     skip_rows = that.get "skip_rows" . if_nothing 0
     row_limit = that.get "row_limit"
     keep_invalid_rows = that.get "keep_invalid_rows" . if_nothing True
+    quote_style = case that.get "quote_style" of
+        Nothing -> Quote_Style.With_Quotes
+        json -> Quote_Style.from json
```

Comment:
```suggestion
    quote_style = if that.get "quote_style" . is_nothing then Quote_Style.With_Quotes else Quote_Style.from json
```

Reply:
This can't work - `json` is undefined.

Reply:
It'd have to be
```ruby
quote_style = if that.get "quote_style" . is_nothing then Quote_Style.With_Quotes else Quote_Style.from (that.get "quote_style")
```
but then we are duplicating `that.get "quote_style"` which I think looks bad. The whole purpose of the `case of` was to write `that.get "quote_style"` once but avoid giving it a separate name (I don't like temporary variables like `quote_style_json` next to the actual `quote_style` and case of allows to hide that)